dds/DCPS/DomainParticipantFactoryImpl.cpp

```diff
@@ -100,7 +100,7 @@ DomainParticipantFactoryImpl::create_participant(
   // needed for the built in topics during enable.
   dp->set_object_reference(dp_obj);  //xxx no change
 
-  if (qos_.entity_factory.autoenable_created_entities == 1) {
+  if (qos_.entity_factory.autoenable_created_entities == true) {
```

Comment:
In a lot of existing places OpenDDS uses ==true and ==false, I just tried to keep the code consistent. If you want to remove ==false/==true I am in favor but that should be a general cleanup action